setup.cfg

```diff
@@ -1,23 +1,92 @@
+
+[metadata]
+name = graphql-ws
+version = 0.3.1
+description = Websocket server for GraphQL subscriptions
+long_description = file: README.rst, CHANGES.rst
+author = Syrus Akbary
+author_email = me@syrusakbary.com
+url = https://github.com/graphql-python/graphql-ws
+keywords = graphql, subscriptions, graphene, websockets
+license = MIT
+classifiers =
+    Development Status :: 2 - Pre-Alpha
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Natural Language :: English
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.6
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.3
+    Programming Language :: Python :: 3.4
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
```

Comment:
wasn't it better to keep this kind of information separate on the `setup.py` file and keep only formatting config on this file?

Reply:
eh, I dunno it's all bikeshed painting. I like having nothing in my setup.py personally (eventually we won't need it, hopefully :D)